examples/java/README.md

```diff
@@ -2,7 +2,7 @@
 
 Ensure that you have an instance of Valkey running on "localhost" on "6379". Otherwise, update glide.examples.ExamplesApp with a configuration that matches your server settings.
 
-To run the example:
+To run the examples for standalone and cluster:
```

Comment:
How can a user run only one of standalone/cluster? most users have only standalone or only cluster setups, they don't need to run both. I think the grade build should be separate. e.g.:
```
To run a standalone example:
 ./gradlew :run-standalone
To run a cluster example for:
 ./gradlew :run-cluster
.

Reply:
```
./gradlew :run --args="standalone"
./gradlew :run --args="cluster"
```
There is one app, but differernt examples are being run depending on the command line arguments

Reply:
I decided to remove ExamplesApp so that user can decide which application to run.